docs/source-fabric/api/collectives.rst

```diff
@@ -0,0 +1,20 @@
+.. include:: ../links.rst
+
+####################################
+lightning.fabric.plugins.collectives
+####################################
+
+
+Collectives
```

Comment:
(pre-existing issue) This is very WIP and experimental. Can you add an "experimental" notice?

```python
    .. warning::  This is an :ref:`experimental <versioning:Experimental API>` feature.
```